packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/run-transaction-endpoint-v1.test.ts

```diff
@@ -251,6 +251,29 @@ describe(testCase, () => {
     }
 
     {
+      const connectionProfileBase64Encoded = Buffer.from(
```

Comment:
@jagpreetsinghsasan Please try to avoid chaining method calls so that if there is a crash with a line number there is no chance of ambiguity where we would not be able to know for sure which method invocation crashed.